tests/testthat/ref/ex1.log

```diff
@@ -0,0 +1,132 @@
+--------------------------------------------------------------------------------
+-                               timber Metadata                                -
+--------------------------------------------------------------------------------
+This log was generated using timber 0.0.0.9002
+timber package version: 0.0.0.9002
+timber build: local
+timber link to repository: https://github.com/atorus-research/timber
+--------------------------------------------------------------------------------
+-                               User Information                               -
+--------------------------------------------------------------------------------
+User: nmasel
+--------------------------------------------------------------------------------
+-                             Session Information                              -
+--------------------------------------------------------------------------------
+─ Session info  ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+ hash: family: man, man, girl, girl, rolled-up newspaper, Ophiuchus
```

Comment:
The random emoji hash is removed in the latest version of [sessioninfo](https://github.com/r-lib/sessioninfo/commit/1f38eeaba6fdbf292340199b443b60d2613d0409).  Do we want to upgrate sessioninfo and rerun these examples to update the log?  I'm ok either way.